packages/errors/src/messages.ts

```diff
@@ -402,6 +403,7 @@ export const SolanaErrorMessages: Readonly<{
         'Expected base58-encoded signature to decode to a byte array of length 64. Actual length: $actualLength.',
     [SOLANA_ERROR__KEYS__SIGNATURE_STRING_LENGTH_OUT_OF_RANGE]:
         'Expected base58-encoded signature string of length in the range [64, 88]. Actual length: $actualLength.',
+    [SOLANA_ERROR__KEYS__VERIFY_SIGNATURE_FAILURE]: 'The provided private key and public key are not a valid key pair',
```

Comment:
We should probably provide a context for that error as well, even if that only includes the public key.

Then this message can be something like:

```ts
'The provided private key does not match the provided public key at address [$address].'
```